assets/src/components/SelectCard.js

```diff
@@ -117,23 +136,54 @@ const SelectCard = props => {
 
   function getCardImage (cardData) {
     if (cardData && cardData.image) {
-      return <CardMedia className={classes.media} image={cardData.image} title={cardData.title} />
+      return (
+        <>
+          <Link className={classes.viewLink} tabIndex={-1} to={cardData.path}>
+            <CardMedia className={classes.media} image={cardData.image} title={cardData.title} />
+          </Link>
+        </>
+      )
     } else {
       return null
     }
   }
 
+  function getHelpLink (cardData) {
+    if (cardData.helpUrl) {
+      return (
+        <Typography gutterBottom variant='h5' component='h4' className={classes.title}>
+          <Tooltip title={'About ' + cardData.title}>
+            <MUILink className={classes.infoLink} href={cardData.helpUrl} target='_blank' rel='noopener noreferrer'>
+              <InfoIcon />
```

Comment:
Nit-pick: we could make this icon a little bigger to fill the space, be more noticeable.

Reply:
I can set it to 'large' size, not sure if there is more flexibility than 3 sizes.
Is this better? @ssciolla @jennlove-um 
![image](https://user-images.githubusercontent.com/17617998/90273924-b7fe1c80-de2d-11ea-8be8-61bea7563bc5.png)

Reply:
Vs
![image](https://user-images.githubusercontent.com/17617998/90274138-0ca19780-de2e-11ea-9d93-6f1018b8f291.png)

Reply:
I like the larger size better.

Reply:
Switched to the larger icon